.github/workflows/vulnerability_scan_subworkflow.yml

```diff
@@ -11,12 +11,15 @@ on:
             ref:
                 required: true
                 type: string
+        secrets:
+            SNYK_TOKEN:
+                required: true
```

Comment:
could we use `${{ secrets.SNYK_TOKEN }}` for `workflow_dispatch` ?

Reply:
not sure about it, this method is mentioned in GitHub docs

Reply:
https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow
AFAICS, we can use secrets param only for `workflow_call`.
For `workflow_dispatch`, secrets are available as per org/repo settings.